programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -198,7 +262,10 @@ static void on_cmd(uv_stream_t *s, ssize_t len, const uv_buf_t *b) {
 
         tunnel_comand cmd = {0};
         if (parse_tunnel_comand(&cmd, b->base, len) > 0) {
-            CMD_CTRL->process(&cmd, on_command_resp, s);
+            int status = process_tunnel_commands(&cmd, on_command_resp, s);
```

Comment:
I'm curious, why are some commands handled in `process_tunnel_commands` and others in `process_cmd`?

Reply:
process_tunnel_commands is used to update the log level and the tun ip information in the config file through IPC command. So when the user restarts the tunnel, the new values will be taken. The config file can be modified only from ziti-edge-tunnel.c file.

process_cmd will delegate the requests to ziti_tunnel_ctrl.c , which is used to perform the operations against the controller. config.file cannot be modified or read from that class

Reply:
this would be a great comment in the code.

Reply:
updated the comment